src/problem.jl

```diff
@@ -213,6 +213,7 @@ end
 make_kwarg(; kwargs...) = kwargs
 
 function JumpProblem(prob, aggregator::AbstractAggregatorAlgorithm, jumps::JumpSet;
+        vr_aggregator::VariableRateAggregator = VRDirectCB(),
```

Comment:
```suggestion
        vr_aggregator::VariableRateAggregator = VRFRMODE(),
```

I'd be willing to just change this default and merge as-is @isaacsas

Reply:
We can make an issue to follow up with performance optimizations and benchmarking to change the default.